### DISCUSSION TITLE
Best way to play music on local speakers

### DISCUSSION THREAD

**Discussion by dapuzz** [music-assistant/support#1911] [Q&A] [answered]
Hi all, sorry for dumb question... what is the **best option to play on local** device? 
Using Snapserver/ Snapclient? May be squeezelite as in FAQ? Or VLC provided by homeassistant? 
Can we discuss about pro and cons of each architecture?

Thank you

**Comment by OzGav** [comment]
Local to what? What is missing in the FAQ?

**Reply by dapuzz** [reply]
How to use the speakers of the mini pc where hass music assistant server is running.

Or another raspberry audio device in the lan...

**Reply by OzGav** [reply, marked as answer]
Squeezelite addon for the HA host. picoreplayers on other RPIs
